src/utils/db.js

```diff
@@ -0,0 +1,16 @@
+const { Sequelize } = require('sequelize');
+const { config } = require('dotenv');
+
+config();
+
+const client = new Sequelize({
+  host: process.env.DB_HOST,
+  username: process.env.DB_USER,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_DATABASE,
```

Comment:
Ensure that `process.env.DB_HOST`, `process.env.DB_USER`, `process.env.DB_PASSWORD`, and `process.env.DB_DATABASE` are correctly set in your environment to avoid issues with the database connection.